frontend/src/v4/routes/viewerGui/components/tree/components/treeNode/treeNode.styles.ts

```diff
@@ -138,6 +159,17 @@ export const Container = styled.li<IContainer>`
 	box-sizing: border-box;
 	cursor: inherit;
 
+	${({ $truncateLeft }) => $truncateLeft && css`
```

Comment:
This doesn't seem to be truncating left
![image](https://github.com/3drepo/3drepo.io/assets/62481250/48d27d0c-bff3-4387-9db3-8b0080271ae6)
I take it the idea is that the root node is displayed in full and children are truncated at the start since the distinguishing information is usually at the end of the name?

Reply:
truncateLeft was a leftover name, I renamed it to be more meaningful, i.e. "isContainer".

The idea is to display the full name of the container(s), not the root.
In your case, I assume you are looking at a federation that has at least 2 containers "Cathedral Door" and "incredibly long ... sort of thing". If you open the respective containers in the viewer, those very same name should be truncated